### PR TITLE
documentation for selecting a pod, cluster or host during VM deploy

### PR DESCRIPTION
**Pull request by houthuis**

### CODE REVIEW

File: source/adminguide/virtual_machines.rst [hunk 1]
```diff
@@ -185,7 +185,7 @@ To create a VM from a template:
 
 #. Click Add Instance.
 
-#. Select a zone.
+#. Select a zone. Admin users also have to option to select a pod, cluster or host.
```
Comment (rafaelweingartner) [inline source/adminguide/virtual_machines.rst]: do you mean via API (Cloudmonkey or any other tool), right?

**Comment by rhtyd** [comment]
@borisstoyanov @PaulAngus - can you also review?
I'll address them changes.

File: source/adminguide/virtual_machines.rst (hunk 1 above)
Comment (rhtyd) [reply source/adminguide/virtual_machines.rst]: The list if wrt UI. See lines above line nu. 185.

**Comment by PaulAngus** [comment]
LGTM

**Pull request merged**